### DISCUSSION TITLE
How to remove default chevron icron in <summary> tag when using dropdown?

### DISCUSSION THREAD

**Discussion by JosephKorel** [saadeghi/daisyui#2511] [Q&A] [answered]
Hi, i have one dropdown nested inside another, this is the child dropdown

```
    <details className="dropdown dropdown-left dropdown-end">
      <summary>Open Second</summary>
      <ul className="p-2 shadow menu dropdown-content z-[1] bg-base-100 rounded-box w-52">
        <li>
          <a>Item 1</a>
        </li>
        <li>
          <a>Item 2</a>
        </li>
      </ul>
    </details>
```
It's working fine, but i don't want that chevron icon to appear. Is there a workaround to that?

**Comment by saadeghi** [comment, marked as answer]
If you use a daisyUI button class for the `<summary>` like this:  `<summary class="btn">` the chevron will go away, otherwise, daisyUI doesn't apply styles to it:   
https://play.tailwindcss.com/Y5naFVQh4W

Another way is to add this arbitrary class: `[list-style:none]` to `<summary>`  
https://play.tailwindcss.com/fb7glrAlfT

**Comment by ankitoye** [comment]
Cleaner way `<summary className="after:content-none">{item.name}</summary>`
